docs/prover/cli/options.md

```diff
@@ -328,6 +328,50 @@ The default number of loop iterations we unroll is one. However, in many cases,
 certoraRun Bank.sol --verify Bank:Bank.spec --loop_iter 2
 ```
 
+Options regarding hashing of unbounded data
+-------------------------------------------
+
+(--optimistic_hashing)=
+### `--optimistic_hashing`
+
+**What does it do?**
+
+ When hashing data of potentially unbounded length (including unbounded arrays, like `bytes`, `uint[]`, etc.), assume that its length is bounded by the value set through the `--hashing_length_bound` option. If this is not set, and the length can be exceeded by the input program, the prover reports an assertion violation. I.e., when this option is set, the boundedness of the hashed data assumed checked by the prover, when this option is set that boundedness is assumed instead.
+
+**When to use it?**  
+
+When the assertion regarding unbounded hashing is thrown, but it is acceptable for the prover to ignore cases where the length hashed values exceeds the current bound.
+
+**Example**
+
+```
+certoraRun Bank.sol --verify Bank:Bank.spec --optimistic_hashing
+```
+
+(--hashing_length_bound)=
+### `--hashing_length_bound`
+
+**What does it do?**
+
+Constraint on the maximal length of otherwise unbounded data chunks that are being hashed. This constraint is either assumed or checked by the prover, depending on whether `--optimistic_hashing` has been set. The bound is specified as a number of bytes. 
+
+**When to use it?**  
```

Comment:
What's the default value?

Reply:
I was a bit cautious about including it tbh, since it might get outdated ... (the help shows it) -- but if you think we should take that risk / work load, I'll add it.
(if this was just an informative question: it's 224 bytes / 7 words)

Reply:
I think it makes sense.  Someday we'll have a release process that hopefully would catch these kinds of inconsistencies when we change things... and if not, maybe we'll get a bug report